docs/virtual_machines/startup_scripts.md

```diff
@@ -40,9 +40,12 @@ The general flow is:
         %WINDIR%\system32\sysprep\sysprep.exe /generalize /shutdown /oobe /mode:vm
 
     More information can be found here:
+
     * [Sysprep Process Overview](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep-process-overview)
     * [Sysprep (Generalize) a Windows installation](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation)
 
+    Note: It is important that there is no answer file detected when the Sysprep Tool is triggered, because Windows Setup searches for answer files at the beginning of each configuration pass and caches it. If that happens, when the OS will start - it will just use the cached answer file, ignoring the one we provide through the Sysprep API. More information can be found [here](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/windows-setup-automation-overview#implicit-answer-file-search-order)
```

Comment:
`Note:` Let's get this changed to a real annotation.  See this [page](https://github.com/kubevirt/user-guide/blame/master/docs/web_console.md#L3)

After this update we are ready.  Thanks!

Reply:
See red arrow in img .... 
<img width="740" alt="img" src="https://user-images.githubusercontent.com/2479463/132722595-cb114d50-ebb1-426e-8110-898695b65e76.png">

The section is very busy.  We need some line breaks.  I think this note should be encapsulated into an annotation like the proposed L53.  Use `!!! Note` to engage the annotation feature.

Reply:
@mazzystr is it better now?

Reply:
Looks great.  Thanks.